src/elisp/treemacs-header-line.el

```diff
@@ -69,10 +69,10 @@
                nil
                `(["Edit Workspaces"       treemacs-edit-workspaces]
                  ["Create Workspace"      treemacs-create-workspace]
-                 ["Remove Worspace"       treemacs-remove-workspace]
```

Comment:
The values aligning as before is more important than a minimal diff.

Reply:
Fixed.